internal/pkg/metrics/metrics.go

```diff
@@ -22,6 +23,17 @@ import (
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
+//MetricHandlerHTML contains HTML for default endpoint
+const MetricHandlerHTML = `
+<html>
+		<head><title>Collectd Exporter</title></head>
```

Comment:
Probably should `s/Collectd/Metrics/` in this block, or have it actually driven by the datasource name